idaes/apps/grid_integration/tests/test_PEM_Parameterized_bidder.py

```diff
@@ -0,0 +1,72 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory,
+# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
+# University, West Virginia University Research Corporation, et al.
+# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
+# for full copyright and license information.
+#################################################################################
+
+import pytest
+import pyomo.environ as pyo
+from idaes.apps.grid_integration.bidder import PEMParametrizedBidder
+from idaes.apps.grid_integration.forecaster import PerfectForecaster
+from idaes.apps.grid_integration.tests.util import (
+    TestingModel,
+    TestingForecaster,
+    testing_model_data,
+)
+from pyomo.common import unittest as pyo_unittest

```

Comment:
Group imports together, starting with native Python packages, then third-party packages, then Pyomo and finally IDAES.